payment/template.yaml

```
@@ -258,4 +258,391 @@ Resources:
   DeadLetterQueue:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: ../../shared/templates/dlq.yaml
\ No newline at end of file
+      TemplateURL: ../../shared/templates/dlq.yaml
+
+ #############
+  # DASHBOARD #
+  #############
+  Dashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+      DashboardName: !Ref AWS::StackName
+      DashboardBody: !Sub |
+        {
+          "start": "-PT6H",
+          "periodOverride": "inherit",
+          "widgets": [
+            {
+              "type": "text",
+              "x": 0,
+              "y": 0,
+              "width": 20,
+              "height": 1,
+              "properties": {
+                "markdown": "\n# Key Metrics\n"
+              }
+            },
+            {
+              "type": "text",
+              "x": 20,
+              "y": 0,
+              "width": 4,
+              "height": 7,
+              "properties": {
+                "markdown": "\n## Useful links\n\n__CloudWatch Logs Insights queries__:\n\n* [Last 100 warning messages](https://${AWS::Region}.console.aws.amazon.com/cloudwatch/home?region=${AWS::Region}#logs-insights:queryDetail=~\\(end~0~start~-21600~timeType~'RELATIVE~unit~'seconds~editorString~'fields*20timestamp*2c*20level*2c*20message.orderId*20as*20orderId*2c*20message.paymentToken*20as*20paymentToken*2c*20message.message*20as*20message*0a*7c*20filter*20level*20*3d*3d*20*22WARNING*22*0a*7c*20sort*20*40timestamp*20desc*0a*7c*20limit*2020*0a~isLiveTail~false~queryId~'38d167e2-e680-4dc3-80fc-411dbd93a317~source~(~'*2faws*2flambda*2f${OnCreatedFunction}~'*2faws*2flambda*2f${OnModifiedFunction}~'*2faws*2flambda*2f${OnFailedFunction}~'*2faws*2flambda*2f${OnCompletedFunction}~'*2faws*2flambda*2f${ValidateFunction})\\))\n\n__Other links__:\n\n* [Repository](https://github.com/aws-samples/aws-serverless-ecommerce-platform/tree/master/payment)\n"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 1,
+              "width": 10,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  ["ecommerce.payment", "paymentCreated", "environment", "${Environment}", {"color": "#29b6f6", "label": "Payments Created"}],
+                  ["ecommerce.payment", "paymentCompleted", "environment", "${Environment}", {"color": "#66bb6a", "label": "Payments Completed"}],
+                  ["ecommerce.payment", "paymentCanceled", "environment", "${Environment}", {"color": "#ef5350", "label": "Payments Canceled"}]
```

To stay consistent with the previous comments

```suggestion
                  ["ecommerce.payment", "paymentProcessed", "environment", "${Environment}", {"color": "#66bb6a", "label": "Payments Processed"}],
                  ["ecommerce.payment", "paymentCancelled", "environment", "${Environment}", {"color": "#ef5350", "label": "Payments Cancelled"}]
```